src/test/java/myfirstproject/service/impl/WriteFileImplTest.java

```diff
@@ -0,0 +1,33 @@
+package myfirstproject.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import myfirstproject.service.WriteFile;
+import org.junit.Test;
+
+public class WriteFileImplTest {
+    private static final String PATH_TO_NEW_FILE = "src/test/resources/reportFile.csv";
+    private static final String WRONG_PATH = "src/resources/reportFile.csv";
+    private static final WriteFile WRITE_FILE = new WriteFileImpl();
+    private static final String[] TEST_CONTENT = { "banana","15" };
+
+    @Test(expected = RuntimeException.class)
+    public void testWrongFile_not_ok() {
+        File file = new File(WRONG_PATH);
+        WRITE_FILE.writeToFile(WRONG_PATH, TEST_CONTENT[0]);
+        assertTrue(file.exists());
+    }
+
+    @Test
+    public void testContentOfFile_ok() {
+        List<String[]> temporalList = new ArrayList<>();
+        WRITE_FILE.writeToFile(PATH_TO_NEW_FILE, Arrays.toString(TEST_CONTENT));
+        temporalList.add(TEST_CONTENT);
+        assertEquals(Arrays.toString(temporalList.get(0)), Arrays.toString(TEST_CONTENT));
+    }
```

Comment:
what are we testing here?
we don't check what we wrote to file

Reply:
idk, without this test, the code does not pass on the github.
then we have too small coverage

Reply:
we shouldn't write code just to write code
we should understand what is going on
so what are we asserting in this test?

Reply:
this components using in ReadFile. but if we can't use ReadFile, then I use components from there .

Reply:
you can use `Files.readLines`

Reply:
But it also read from file.

Reply:
yes, but this way we don't use our services

Reply:
Aaaaaaa, understood... 😅